packages/evolution-frontend/src/components/inputs/InputCheckbox.tsx

```diff
@@ -184,11 +184,10 @@ export class InputCheckbox<CustomSurvey, CustomHousehold, CustomHome, CustomPers
     };
 
     private getColumnedChoices = (choiceInputs: JSX.Element[]): JSX.Element[] => {
-        if (this.props.widgetConfig.columns === undefined || this.props.widgetConfig.columns <= 0) {
+        if ((this.props.widgetConfig.alignChoices === undefined || !this.props.widgetConfig.alignChoices)) {
```

Comment:
Here, check columns, rows and customAlignmentLength instead, to drop alignChoices. This check is probably the cause of the test failure, as this widgetConfig is not set and the columns are thus not considered anymore